ui/component/app/view.jsx

```diff
@@ -141,6 +141,7 @@ function App(props: Props) {
   const [localeSwitchDismissed] = usePersistedState('locale-switch-dismissed', false);
   const [showAnalyticsNag, setShowAnalyticsNag] = usePersistedState('analytics-nag', true);
   const [lbryTvApiStatus, setLbryTvApiStatus] = useState(STATUS_OK);
+  usePersistedState('watch-history', []);
```

Comment:
I had to pre-create it there because the player itself constantly gets destroyed/reinitialized which makes the event-listener for the persistent state useless. Feel free to adjust it if there's a better solution for this.

Reply:
It probably won't matter now when we move to redux.

Reply:
On closer look, it seems like `selectHistory`, `doSetContentHistoryItem`, `doClearContentHistoryUri`, and `doClearContentHistoryAll` covered everything, so we can probably skip all the `setWatchHistory` stuff.